src/2025h1/seamless-rust-cpp.md

```diff
@@ -39,7 +39,7 @@ Roughly speaking, there are three axes to adoption of memory safety: Social, Tec
 
 For example, safety has become more socially desirable in many technical communities over the years, which has led to the development of mitigation measures and the adoption of languages like Rust. This has come partly as a result of the recognition of the economic costs of memory safety vulnerabilities.
 
-For C/C++ this has led to an improvement along the technical front in terms of automated checking, in both static and dynamic tooling. However, this protracted effort has also revealed the limits of such an approach without language changes. While there have been growing calls for C++ to adopt memory safety features,[^safe-cpp] they have not gained traction within the body that controls the C++ standard for a combination of structural, technical, social, and economic reasons.[^corentin-profiles] The cost to adopting such features in C++ is too high for a combination of technical, social, and economic reasons.
+For C/C++ this has led to an improvement along the technical front in terms of automated checking, in both static and dynamic tooling. However, this protracted effort has also revealed the limits of such an approach without language changes. While there have been calls for C++ to adopt memory safety features,[^safe-cpp] they have not gained traction within the C++ standards body for a combination of technical, social, and economic reasons.[^corentin-profiles]
```

Comment:
I think some would argue that [safety profiles](https://isocpp.org/files/papers/P3081R0.pdf) are addressing the safety issues, but the approach is fundamentally different to Rust, so perhaps it would be more accurate to say that the standards body has reviewed a proposal for evolving the language to add a Rust-like safe-by-construction subset and is [currently opting to pursue a previously proposed, less-invasive alternative](https://www.youtube.com/watch?v=4yphbBBj5sQ) in an attempt to speed and simplify adoption, but which does not provide guarantees against undefined behavior or data races that are core to Rust's value proposition.